### PR TITLE
Remove description prefixes from translated strings (master)

### PR DESCRIPTION
**Pull request by Wedge009**
master version of #4271.

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
I wonder if we can somehow make pot-update do this automatically?

**Comment by stevecotton** [comment]
It can't be automatic - Wedge009 has reviewed each one and worked out whether to remove only the prefix, or whether the whole line is wrong and needs to be considered as untranslated.

File: po/wesnoth/nl.po [hunk 1]
```diff
@@ -7188,15 +7188,15 @@ msgstr "Schakel een kant naar/van AI controle."
 #, fuzzy
 #| msgid "do not translate the on/off^[<side> [on/off]]"
 msgid "do not translate the on/off/full^[<side> [on/off/full]]"
-msgstr "vertaal de on/off^[<zijde>[on/off]] niet"
+msgstr "[<zijde>[on/off]]"
```
Comment (stevecotton) [inline po/wesnoth/nl.po]: This is missing the 'full', which seems a common problem for translations of this string (although most don't have the prefix).

What can we do to work better with the translators? It's clear that a lot of fuzzy strings have been simply marked as up-to-date.

**Comment by Wedge009** [comment]
I'm out on Sundays but can review the comments you made with machine translation's help before you merge, if you prefer. (23:42 UTC as I write this - am normally back around 12:00 UTC)

File: po/wesnoth/nl.po (hunk 1 above)
Comment (Wedge009) [reply po/wesnoth/nl.po]: Another case of a fuzzy translation that needs to be updated. Leaving as-is.

**Pull request merged**